lib/components/normal-components/Resonator.ts

```diff
@@ -0,0 +1,47 @@
+import { resonatorProps } from "@tscircuit/props"
+import { NormalComponent } from "../base-components/NormalComponent"
+import type { BaseSymbolName } from "lib/utils/constants"
+
+export class Resonator extends NormalComponent<typeof resonatorProps> {
+  getResonatorSymbolName = (variant: string | undefined): BaseSymbolName => {
```

Comment:
this is even worse! it needs to be in the root scope. You've added a new public property to resonators.

You could theoretically do this if it was internal, but then you'd have to change the function